src/cli/help.js

```diff
@@ -55,7 +55,6 @@ function usage(print: (string) => void): void {
       AUXILIARY
       load          load plugin data into cache
       graph         build Cred graph from cached plugin data
-      score         generate Cred Scores from from graph
```

Comment:
let's just update this to be `credrank` instead of `score`.  The description is still accurate.

Reply:
https://github.com/sourcecred/sourcecred/pull/2839